lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1502,6 +1503,23 @@ def _calc_coord(self, xv, yv, renderer=None):
         p2 = p1 - scale*vec
         return p2, pane_idx
 
+    def _arcball(self, p):
```

Comment:
```suggestion
    def _arcball(self, x: float, y: float) -> np.ndarray:
```
While, `mplot3d` is historically untyped, it makes sense to include type hints for new code.

Since _arcball is internal, we can freely choose it's interface and for it's only usages, a component approach seems simpler.

Reply:
Makes sense, thank you for the suggestion.